frontend/src/metabase-types/api/user.ts

```diff
@@ -31,6 +31,7 @@ export interface User extends BaseUser {
   custom_homepage: {
     dashboard_id: DashboardId;
   } | null;
+  dismissed_toasts: Record<string, boolean>;
```

Comment:
Why do we need this? It seems that we can only use the setting, which is user-local